integration/image/Dockerfile

```diff
@@ -1,16 +1,33 @@
-# Generated on: Wed Jul  8 11:06:05 IST 2020
+# Generated on: Thu 13 Aug 10:09:50 IST 2020
 
 # Generated by: wf-gen from tyk-ci
 
 FROM debian:buster-slim
 
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get install -y --no-install-recommends \
+        wget jq curl ca-certificates apt-transport-https gnupg unzip
+
+RUN apt-get install -y --no-install-recommends \
+        build-essential \
+        python3-setuptools \
+        libpython3.7 \
+        python3.7-dev \
+        python3-pip \
+        && pip3 install protobuf grpcio==1.24.0 \
+        && apt-get purge -y build-essential \
+        && rm -rf /root/.cache
+RUN apt-get autoremove -y 
+
 ADD tyk.tar.gz /opt/tyk
```

Comment:
Why is this ADD operation needed? If I try to build the image I'm getting an error: 
`Step 5/9 : ADD tyk.tar.gz /opt/tyk`
`ADD failed: stat /var/lib/docker/tmp/docker-builder541896976/tyk.tar.gz: no such file or directory`

Reply:
The tarfile is built by https://github.com/TykTechnologies/tyk/blob/master/bin/integration_build.sh and the image is built and pushed in the int-image workflow. This image is meant to be built by automation.